_docs/create-bitcoin-wallet.md

```diff
@@ -0,0 +1,85 @@
+---
+title: How to create a Bitcoin wallet
+tags: [setup]
+---
+
+Creating a Bitcoin wallet on BlueWallet is easy.
+
+You can either click on the "Add now" button or the "+" icon in the top right corner
+
+{% include phone-screenshot-image.html img="bluewallet-first-page.png" alt="BlueWallet's first page" %}
+
+That will take you to the "add wallet" screen.
+
+## Adding a new wallet
+
+In the "add wallet" screen, you can choose between a Bitcoin wallet and a Lightning wallet.
+
+{% include phone-screenshot-image.html img="add-bitcoin-wallet.png" alt="Add bitcoin wallet page" %}
+
+For Bitcoin, click on the Bitcoin button. (If you want to create a Lightning wallet, see [this guide](/docs/create-lightning-wallet).)
+
+You may also name your wallet, or leave the default name.
+
+<div class="side-note" markdown="1">
+
+### Advanced options
+
+If you have "Advanced mode" enabled in general settings,
+you'll see more options for creating the wallet.
+
+{% include phone-screenshot-image.html img="add-bitcoin-wallet-with-advanced-options.png" alt="Add Bitcoin wallet with advanced options" %}
+
+#### Wallet type
+
+The first group of options is for the wallet type:
```

Comment:
I'd like to add more details about the different types. Why would the user choose one type over the other? Are there trade offs? Or is it for compatibility with other wallets?

Reply:
HD means wallet uses seed to derive keys from it using some algorithm
https://en.bitcoin.it/wiki/Deterministic_wallet 

Bech32 - modern type of bitcoin addresses. Bech32 is more efficient with block space. But some old wallets can't send bitcoins to it.
https://en.bitcoin.it/wiki/Bech32

Segwit P2SH  - or wrapped segwit. This wallet generates addresses starting with `3` They are backwards compatible with old software, more efficient than legacy addresses, but less that bech32.
https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#P2SH-wrapped_segwit

HD Segwit BIP49 - HD version 

I know that Segwit P2SH and HD Segwit BIP49 lacks some features, like send to a few addresses in one transaction

Reply:
@limpbrains why would the user choose SegWit P2SH instead of HD SegWit BIP49? From what I understand, the only difference between the two is that one is HD the other isn't? Wouldn't HD always be the better option?

Reply:
I think HD is always a better option. I don't know why it is possible to create no HD wallet here, let's ask @ncoelho or @Overtorment

Reply:
its a leftover from times when BW didnt support HD. anyways, its hidden under "advanced mode", so only people who know what they are doing should use it.

Reply:
BIP84 is the best (and default) format